python/cuml/dask/datasets/blobs.py

```diff
@@ -79,6 +88,8 @@ def make_blobs(nrows, ncols, centers=8, n_parts=None, cluster_std=1.0,
     :param random_state : sets random seed
     :param verbose : enables / disables verbose printing.
     :param dtype : (default = np.float32) datatype to generate
+    :param output : (default = 'dataframe') whether to generate dask array or
+    dask dataframe output. Default will be array soon.
```

Comment:
I think that is a fantastic idea! Probably a follow up PR, will open an issue since it sounds like a good first issue for someone wanting to get familiar with cuml.dask